### ISSUE TITLE
wrong multiple monitors arrangement

### ISSUE THREAD

**Issue by kyuwoo-choi** [TigerVNC/tigervnc#1617]
**Describe the bug**

**To Reproduce**
1. Check windows 11 "System > Display" setting
2. Take a look at the monitor arrangement
3. Start vncserver 1.12.0 on Ubuntu 22.10
4. Start VNC Viewer windows with vncviewer64-1.13.1.exe
5. Select the server uri
6. Select "Options" button
7. Select "Display" tab
8. Select "Full screen on all monitors"
9. Take a look at the monitor arrangement, which is different from the Windows settings.
10. Close the "Options" pane, Select "Connect"
11. Desktop screens are wrong as the monitor arrangement

**Expected behavior**
TigerVNC display mode for all monitors should reflect the windows display arrangement setting.

**Screenshots**
#### windows display setting
<img width="639" alt="image" src="https://user-images.githubusercontent.com/1215767/231009360-75e35007-0870-4214-b5c3-21977d044f06.png">

#### vnc client display mode
<img width="340" alt="image" src="https://user-images.githubusercontent.com/1215767/231009386-07a15576-4b3e-40f4-aff5-79bcfbf2335c.png">

#### vnc screens
<img width="4280" alt="image" src="https://user-images.githubusercontent.com/1215767/231010226-b8454f8d-ec08-4ce4-9400-c40143e80fad.png">

**Client (please complete the following information):**
 - OS: Windows 11
 - VNC client: TigerVNC
 - VNC client version: 1.13.1
 - Client downloaded from: https://sourceforge.net/projects/tigervnc/files/stable/1.13.1/

**Server (please complete the following information):**
 - OS: Windows 11 with WSL2
 - VNC server: TigerVNC 
 - VNC server version: 1.12.0
 - Server downloaded from: Ubuntu repo via apt
 - Server was started using: vncserver

**Additional context**

**Comment by kyuwoo-choi** [comment]
I just figured out why.
It was because of inconsistent monitor scales.
My previous setup has 125% scale for monitor 1, 150% for the other monitors.
I changed all of those to 150% and it works good.

**Comment by samhed** [comment]
Yeah, as you noticed, the issue goes beyond the monitor arrangement GUI. The problem has been around for a long time, even before the monitor arrangement GUI came around. However, I couldn't find any open issues for it, so it's good you reported this. The screenshots are useful, thank you.

Unfortunately, I suspect we won't have time to look at a fix right now.

**Comment by CendioOssman** [comment]
This is part of the work needed for #369.

**Comment by sdistefano** [comment]
+1 would love to see this fixed one day.
